CHANGELOG.md

```diff
@@ -3,16 +3,19 @@
 ## v0.4.2
 - Added an `Init` struct, which can help with initial routing (Breaking)
 - The `routes` function now returns an `Option<Msg>` (Breaking)
-- updated `Tag::from()` to accept more input types
+- Updated `Tag::from()` to accept more input types
+- `style!` now accepts types that implement the new `ToCssValue` trait
```

Comment:
`ToCssValue` has been renamed to `ToCSSValueForCSSValue` and documentation is hidden.
So I suggest to change the line to something like 
```
- `style!` now accepts also `Option<impl ToString>`
```
---
I don't know if we want to add public trait `ToCSSValue` so users can implement it for custom types. If we want to discuss it I suggest to create a new issue.